sdk/eventhub/Azure.Messaging.EventHubs/src/Core/BasicRetryPolicy.cs

```diff
@@ -20,7 +20,7 @@ internal class BasicRetryPolicy : EventHubRetryPolicy
         private static int s_randomSeed = Environment.TickCount;
 
         /// <summary>The random number generator to use for a specific thread.</summary>
-        private static readonly ThreadLocal<Random> s_random = new ThreadLocal<Random>(() => new Random(Interlocked.Increment(ref s_randomSeed)), false);
+        private static readonly ThreadLocal<Random> RandomNumberGenerator = new ThreadLocal<Random>(() => new Random(Interlocked.Increment(ref s_randomSeed)), false);
 
```

Comment:
If we are refactoring the `s_random` name, should also refactor the `s_randomSeed` name? Was the intent to make the variable name more clear that it was a random number generator or to wack off the `s_`?

Reply:
Our standards require static variables to prefix with `s_`.    Consstants and readonly references are `CamelCased` so... the other has to remain.